pandas/tests/frame/methods/test_describe.py

```diff
@@ -298,3 +299,23 @@ def test_describe_percentiles_integer_idx(self):
             ],
         )
         tm.assert_frame_equal(result, expected)
+
+    def test_describe_does_not_raise_error(self):
+        # GH#32409
+        df = pd.DataFrame([{"test": {"a": "1"}}, {"test": {"a": "2"}}])
+        expected = DataFrame(
+            {"test": [2, 2, {"a": "1"}, 1]}, index=["count", "unique", "top", "freq"]
+        )
+        try:
```

Comment:
you don't need the try/except if it fails then pytest automatically raises.

Reply:
Thank you for the info, I've updated this PR and re-requested a review